### DISCUSSION TITLE
Calibration for EU problems

### DISCUSSION THREAD

**Discussion by SeanVella** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#130] [Q&A]
Good morning fellas! I need some help from you professionals.

The issue I have here is that I am in the EU (Malta) and in the setup it asks to use an AC-AC transformer. (9V/12V 500mA output). I checked the power supply I am using and it says it is an AC Adapter 12V 2A max and 30W max. When trying to calibrate the device I keep getting very low numbers and the voltage is always <1 which I am sure is incorrect.

Does that mean my power supply is not sufficient?

Is there anyone who can help me out with this?

**Comment by coopermor** [comment]
It sounds like you're using an AC-DC transformer not an AC-AC transformer. Do you have a link to the transformer you're using?

**Comment by SeanVella** [comment]
![image](https://user-images.githubusercontent.com/11162226/217828865-bde49fdb-b10d-4a8b-bd35-596282c1411b.png)

This is what I have. I tried both 9V and 12V

**Reply by coopermor** [reply]
That is an AC-DC convertor. You need to buy an AC-AC transformer.

**Reply by SeanVella** [reply]
Is it possible to find one on Amazon? I tried looking for one but I do not seem to find anything on Amazon.de or Amazon.co.uk
